lib/pure/strmisc.nim

```diff
@@ -85,6 +86,130 @@ proc rpartition*(s: string, sep: string): (string, string, string)
 
   return partition(s, sep, right = true)
 
+
+since (1, 5):
+  type ParseFloatOptions* = enum  ## Options for `parseFloatThousandSep`.
+    pfLeadingSep,    ## Allow leading separator, like ",9" and similar.
```

Comment:
when is this ever useful?
I've never seen `,9` in the wild, in the rare case someone needs it, he can use a custom pre-processing?